test/checks/navigation/p-as-heading.js

```diff
@@ -141,6 +141,26 @@ describe('p-as-heading', function() {
     );
   });
 
+  it('returns true if the heading elm length is greater than the paragraph elm', function() {
+    var params = checkSetup(
+      '<p id="target">elm1elm1</p>' + '<p>elm2</p>',
```

Comment:
```suggestion
      '<p id="target"><b>elm1elm1</b></p>' + '<p>elm2</p>',
```

Otherwise this isn't proving the case.